pymess/backend/dialer/__init__.py

```diff
@@ -26,7 +26,7 @@ class DialerBackend(BaseBackend):
 
     model = DialerMessage
 
-    def create_message(self, recipient, content, related_objects, tag, template, **kwargs):
+    def create_message(self, recipient, content, related_objects, tag, template, is_predictive=False, **kwargs):
```

Comment:
I extended the previous backend instead of the new one as it's the same service and even both API calls are the same.